src/main/java/org/cloudfoundry/promregator/scanner/ReactiveAppInstanceScanner.java

```diff
@@ -16,6 +16,7 @@
 import org.cloudfoundry.client.v2.spaces.SpaceApplicationSummary;
 import org.cloudfoundry.client.v2.spaces.SpaceResource;
 import org.cloudfoundry.client.v2.routes.Route;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Log4j is on the outs at the moment. 😉

Reply:
Well, that itself isn't the main reason 😁 
And BTW those folks IMHO do the best that they can right now. Instead of slamming our doors into their face, we should be thankful for them to be transparent...

Reply:
I agree. Sorry my joke wasn't intended to be insensetive.

Reply:
no worries. Didn't read it like this; but these poor guys need our support - even if it is only mentally.

Reply:
Ok I used this, because the library was already in the dependencies. And I think log4j was updated in the meanwhile...